vendor/assets/stylesheets/dvl/core/forms.scss

```diff
@@ -431,24 +436,30 @@ form {
 }
 
 @media only screen and (min-width: $lapWidth) {
+  .form_item_horiz legend,
   .form_item_horiz_label,
   .form_item_horiz_input {
     display: inline-block;
     vertical-align: top;
   }
 
+  .form_item_horiz legend {
+    float: left;
+  }
+
+  .form_item_horiz legend,
   .form_item_horiz_label {
     text-align: right;
-    width: 25%;
-    padding-right: $rhythm * 4;
+    width: 30%;
+    padding-right: $lineHeight;
   }
 
   .form_item_horiz_input {
-    width: 75%;
+    width: 70%;
   }
 
   .form_actions.horizontal {
-    padding-left: 24.99999%;
+    padding-left: 30%;
```

Comment:
You don't have to write a commit message immediately when you make the
change. You can wait until you're ready to commit, and then commit
line-by-line. I use GitHub for Mac to do this easily. Does that help?

On Wed, Mar 23, 2016 at 12:04 PM, Josh Rubenoff <notifications@github.com>
wrote:

> In vendor/assets/stylesheets/dvl/core/forms.scss
> <https://github.com/dobtco/dvl-core/pull/243#discussion_r57186751>:
>
> >    }
> >
> >    .form_actions.horizontal {
> > -    padding-left: 24.99999%;
> > +    padding-left: 30%;
>
> I know; I guess I'm wondering why it's worth the cognitive overhead for me
> to do so when I'm just working on something in context, especially when
> it's a one-line change.
>
> —
> You are receiving this because you were assigned.
> Reply to this email directly or view it on GitHub
> <https://github.com/dobtco/dvl-core/pull/243/files/2c0664d134b8e5b0a8bb5e05102b91dd41e07840#r57186751>
>



-- 
Adam Becker
(951) 9-BECKER
@AdamJacobBecker